src/PHPUnit/Comparator.php

```diff
@@ -1,27 +1,33 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Money\PHPUnit;
 
 use Money\Currencies\AggregateCurrencies;
 use Money\Currencies\BitcoinCurrencies;
 use Money\Currencies\ISOCurrencies;
 use Money\Formatter\IntlMoneyFormatter;
 use Money\Money;
+use NumberFormatter;
 use SebastianBergmann\Comparator\ComparisonFailure;
 
+use function assert;
+
 /**
  * The comparator is for comparing Money objects in PHPUnit tests.
  *
  * Add this to your bootstrap file:
  *
  * \SebastianBergmann\Comparator\Factory::getInstance()->register(new \Money\PHPUnit\Comparator());
+ *
+ * @internal do not use within your sources: this comparator is only to be used within the test suite of this library
```

Comment:
Made this `@internal` - it's not supposed to be used by third-parties, and is only really used within our test suite.

It's also not documented, AFAIK